frontend/__tests__/units.spec.js

```diff
@@ -2,7 +2,7 @@ import * as _ from 'lodash-es';
 
 import { units, validate, convertToBaseValue } from '../public/components/utils/units';
 
-describe('units', () => {
+xdescribe('units', () => {
```

Comment:
We should really try to fix these tests. How bad are the failures?

Reply:
@rawagner has fixed the tests :tada: 

He said the problem originates from commit https://github.com/openshift/console/commit/b14d83ba42bb1f749d515cddc697ffc526c98689 (change in `round` function).